src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -1189,12 +1188,43 @@ public struct S
     public int P1 { readonly get; }
     public readonly int P2 { get; }
     public int P3 { readonly get; set; }
-    public int P4 { readonly get; readonly set; } // PROTOTYPE: readonly set on an auto-property should give an error
-    public readonly int P5 { get; set; } // PROTOTYPE: readonly set on an auto-property should give an error
+    public int P4 { readonly get; readonly set; }
+    public readonly int P5 { get; set; }
+    public readonly int P6 { readonly get; }
 }
 ";
```

Comment:
For completeness, perhaps:
```C#
public int P7 { get; readonly set; }
public readonly int P8 { get; readonly set; }
```

Reply:
Sounds good.

---
In reply to: [267979372](https://github.com/dotnet/roslyn/pull/34338#discussion_r267979372) [](ancestors = 267979372)